docs/UserGuide.md

```diff
@@ -99,9 +99,15 @@ Examples:
 
 ### Listing all residents : `list`
 
-Shows a list of all residents in the RC4HDB database.
+Shows a list of all residents in the RC4HDB database. Use the specifiers `/i` for include and `/e` for exclude, followed by the field names of the columns to show or hide. All field names entered are case-insensitive.
```

Comment:
Might want to indicate that if the user entered the same specifiers multiple times, that only the last specifier is taken.